.eslintignore

```diff
@@ -0,0 +1,3 @@
+/lib
```

Comment:
Is there a lib folder?

Reply:
No, I deleted it, because it was empty, and completely forgot I included it into .eslintignore